lua/plainline/init.lua

```diff
@@ -24,6 +24,10 @@ local function full_config(user_config)
       if not preset then
          error(string.format("Inexistent plainline preset: %s", user_config))
       end
+      if preset ~= presets.default then
+        -- Makes all presets fallback to the default configuration
+        setmetatable(preset, {__index=preset})
+      end
```

Comment:
After thinking a bit about this, I've changed my mind about the metatables thing; I think it would be better to use `vim.tbl_deep_extend`, as we do with the user config below. Also, shouldn't `__index` be set to `presets.default` here?

Reply:
Yes it was a silly mistake, thanks for catching it.

I hadn't noticed it does deep extend before, so I'll be removing this code